lib/perl/Genome/Model/Tools/Somatic/ProcessSomaticVariation.pm

```diff
@@ -937,6 +938,39 @@ sub execute {
 
 
 
+  #----------------------------------------------------
+  # add dbsnp/gmaf
+
+  if ($self->evs_maf_file){
+      $self->status_message("==== adding evs maf info ====");
+      if(-s $self->evs_maf_file){
+          my $db_cmd = Genome::Model::Tools::Annotate::AddEvsMaf->create(
```

Comment:
It gets duplication code line 947 - 963. It can be consolidated as below:
for my $anno_file ($snv_file, $indel_file) {
   my $output_file = $anno_file.'.evs';
   my $db_cmd = Genome::Model::Tools::Annotate::AddEvsMaf->create(
        anno_file => $anno_file,
        output_file => $output_file,
        vcf_file => $self->evs_maf_file,
);
   unless ($db_cmd->execute) {
      die $self->error_message("Failed to add evs maf info to file $anno_file.");
   }
}